.github/workflows/docker-image-verify.yaml

```diff
@@ -121,14 +125,19 @@ jobs:
           JIRA_USER_EMAIL: ${{ secrets.JIRA_USER_EMAIL }}
           JIRA_API_TOKEN: ${{ secrets.JIRA_API_TOKEN }}
 
+      - name: Setting repo name
+        if: failure() && steps.scan_image.outputs.vulnerable == 'true'
+        run: echo "REPOSITORY_NAME=$(echo '${{ github.repository }}' | awk -F '/' '{print $2}')" >> $GITHUB_ENV
+        shell: bash
+
       - name: Check Jira issue existence
         if: failure() && steps.scan_image.outputs.vulnerable == 'true'
         id: issue_check
         env:
           JIRA_API_TOKEN: ${{ secrets.JIRA_API_TOKEN }}
         run: |
           JIRA_ISSUES_COUNT="$(jira list \
-            --query '${{ inputs.jira_jql }}' | wc -l)"
+            --query 'creator = "githubot@intergiro.com" and project = ${{ inputs.jira_project }} and issuetype = "${{ inputs.jira_issuetype }}" and status not in (Done, Archived) and created >= "-30d" and labels = ${{ env.REPOSITORY_NAME }}' | wc -l)"
```

Comment:
Please extract creator to variable as well.

Reply:
Done.